Assets/Programming/Scripts/UI/TitleScreen.cs

```diff
@@ -24,7 +25,7 @@ private void Update()
         }
         if (Mouse.current != null)
         {
-            if (Mouse.current.IsPressed())
+            if (Mouse.current.rightButton.isPressed || Mouse.current.leftButton.isPressed)
```

Comment:
I wonder if there's a way to check if any key or mouse input is entered. Should we be able to press any key to continue?